packages/talos_agent/src/api.rs

```diff
@@ -115,21 +115,21 @@ impl TalosAgentBuilder {
     }
 
     /// Build agent instance implemented using shared state between threads.
-    pub fn build(&self) -> Result<Box<TalosAgentType>, String> {
-        let config = &self.kafka_config.clone().expect("Kafka configuration is required");
-        let publisher: Box<PublisherType> = Box::new(KafkaPublisher::new(self.config.agent.clone(), config));
-        let agent = TalosAgentImpl::new(self.config.clone(), self.kafka_config.clone(), publisher);
-        agent.start().unwrap_or_else(|e| panic!("{}", format!("Unable to start agent {}", e)));
-
-        Ok(Box::new(agent))
-    }
+    // pub fn build(&self) -> Result<Box<TalosAgentType>, String> {
```

Comment:
Please ignore this old agent impl. I am removing it in the next PR which will target this branch.